delta/service/src/main/scala/ch/epfl/bluebrain/nexus/delta/service/syntax/package.scala

```diff
@@ -0,0 +1,3 @@
+package ch.epfl.bluebrain.nexus.delta.service
+
+package object syntax extends EventLogSyntax
```

Comment:
This syntax should import all the syntax available in the parent modules, as done [here](https://github.com/BlueBrain/nexus/blob/master/delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/syntax/package.scala#L9) or [here](https://github.com/BlueBrain/nexus/blob/master/delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/syntax/package.scala#L9) so we don't need to import the syntax of each module, but just of the one we are currently working on.

Once done that, you should add this `EventLogSyntax` to the [app module syntax](https://github.com/BlueBrain/nexus/blob/master/delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/syntax/package.scala#L9)

Reply:
I wouldn't add it in the app because if you doing things with the eventLog directly there, you are doing things wrong